package.json

```
@@ -32,17 +32,17 @@
     "redirect"
   ],
   "peerDependencies": {
-    "vite": "^3.0.0"
+    "vite": "^4.0.0"
```

This can be 
```suggestion
    "vite": "^3.0.0 || ^4.0.0"
```
to support Vite versions.